addons/web_tour/static/src/js/tip.js

```diff
@@ -115,6 +115,10 @@ var Tip = Widget.extend({
         this.$el.toggleClass('d-none', !!this.info.hidden);
         this.el.classList.add('o_tooltip_visible');
         core.bus.on("resize", this, _.debounce(function () {
+            if (this.isDestroyed()) {
+                // Because of the debounce, destroy() might have been called in the meantime.
+                return;
+            }
```

Comment:
Extracted the last 2 commits - and the part about the `website_event` tour to https://github.com/odoo/odoo/pull/83573 because the impacted tours did not exist in 14.0.
Rebased this PR to 14.0.
Investigated 13.0, but it did not contain the revived tips problem nor the other minor issues.